app/controllers/concerns/idv_session_concern.rb

```diff
@@ -47,7 +47,7 @@ def redirect_unless_idv_session_user
 
   def redirect_unless_sp_requested_verification
     return if !IdentityConfig.store.idv_sp_required
-    return if idv_session_user.profiles.any?
+    return if idv_session_user&.profiles&.any?
```

Comment:
https://github.com/18F/identity-idp/pull/11852/files#diff-8916fd256ed3f4acc9ae769fcfadf719c198dfcfe351dcbe16d42fb48cd55627L7 should mean there has to be an idv session user?

Reply:
The ticket suggests we should consider removing [this](https://github.com/18F/identity-idp/blob/0cb8aa37d25cde7d0219d52aaebff425c5c58336/app/controllers/idv/socure/document_capture_controller.rb#L21) instead, which seems right.